### PR TITLE
Config file override

### PR DESCRIPTION
**Pull request by Dezzy666**
I created an optional argument for all public static methods so the API can work with multiple google accounts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,21 @@ Also, take a look at the following files for basic examples of how to retrieve c
 
 * `test.php`
 * `test_individual.php`
+
+## Confg file override
+
+Each method has optional argument for config file override. It is useful when you want to use work witch multiple Google accounts at the same time.
```
Comment (DivineOmega) [inline README.md]: Looks like this should be 'with' rather than 'witch'.

File: README.md [hunk 2]
```diff
@@ -29,3 +29,21 @@ Also, take a look at the following files for basic examples of how to retrieve c
 
 * `test.php`
 * `test_individual.php`
+
+## Confg file override
```
Comment (DivineOmega) [inline README.md]: 'Config' rather than 'Confg'

**Comment by DivineOmega** [comment]
Thanks for your contribution. It looks like a really good addition to the library.

Before I merge your pull request:

* I've added a couple of comments above about some documentation typos that should be fixed ideally.
* Could you quickly change the `$custom_config` to be camel-cased in the code and documentation, just for consistency?

Thanks again!

**Comment by Dezzy666** [comment]
That naming convention makes sense, I should have noticed. Thanks.

**Comment by DivineOmega** [comment]
Looks awesome. I'll merge this. :)

Thanks for contributing.

**Pull request merged**